models/team_todolist.js

```diff
@@ -0,0 +1,43 @@
+module.exports = (sequelize, DataTypes) => {
+    const team_todolist = sequelize.define("team_todolist", {
+        todolist_ID: {
+            type: DataTypes.STRING,
+            primaryKey: true,
+            allowNull: false
+        },
+        team_ID:{
+            type: DataTypes.STRING,
+            allowNull: true
+        },
+        name: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        description: {
+            type: DataTypes.STRING,
+            allowNull: false
+        },
+        state:{
+            type: DataTypes.INTEGER,
+            allowNull: false
+        },
+        todolist_index:{
+            type: DataTypes.INTEGER,
+            allowNull: false
+        },
+        expire_datetime:{
+            type: DataTypes.DATE,
+            allowNull: true
+        },
+        start_datetime: {
+            type: DataTypes.DATE,
+            allowNull: false
+        }
+    },{freezeTableName: true, timestamps: false})
+
+    team_todolist.associate = models => {
+        team_todolist.belongsTo(models.team, {foreignKey:"team_ID"})
+    }
```

Comment:
I think it have associate with `todolist` model too.

Reply:
Relation ship from team to todolist is 1:M, so I make team_todolist a seperate todolist table, should I merg it with normal todolist?

Reply:
so you mean it was same model of `todolist` but using for team?

Reply:
Yes, there is only one difference is it keep 'team_id'​ instead of 'user_id'.